src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/ptv/AbstractPtvInTest.java

```diff
@@ -0,0 +1,240 @@
+    return Arrays.asList(result);
+  }
+
+  protected List<VmOpenApiLocalizedListItem> createPtvInLocalizedItems(String language, String type, String value) {
+    VmOpenApiLocalizedListItem result = new VmOpenApiLocalizedListItem();
+    result.setLanguage(language);
+    result.setType(type);
+    result.setValue(value);
+    return Arrays.asList(result);
+  }
+
+  protected List<VmOpenApiLanguageItem> createPtvInLanguageItems(String language, String value) {
+    VmOpenApiLanguageItem result = new VmOpenApiLanguageItem();
+    result.setLanguage(language);
+    result.setValue(value);
+    return Arrays.asList(result);
+  }
+  
+  protected List<LocalizedValue> createLocalizedValue(String language, String value) {
+    return createLocalizedValue(language, null, value);
+  }
+
+  protected List<LocalizedValue> createLocalizedValue(String language, String type, String value) {
+    LocalizedValue result = new LocalizedValue();
+    result.setLanguage(language);
+    result.setType(type);
+    result.setValue(value);
+    return Arrays.asList(result);
+  }
+
+  protected V4VmOpenApiServiceHour creaatePtvInServiceHour(Boolean isClosed, List<V2VmOpenApiDailyOpeningTime> openingHour, String serviceHourType, Boolean validForNow, OffsetDateTime validFrom, OffsetDateTime validTo, List<VmOpenApiLanguageItem> additionalInformation) {
+    V4VmOpenApiServiceHour result = new V4VmOpenApiServiceHour();
+    result.setAdditionalInformation(additionalInformation);
+    result.setIsClosed(isClosed);
+    result.setOpeningHour(openingHour);
+    result.setServiceHourType(serviceHourType);
+    result.setValidForNow(validForNow);
+    result.setValidFrom(validFrom);
+    result.setValidTo(validTo);
+    return result;
+  }
+
+  protected ServiceHour createServiceHour(Boolean isClosed, List<DailyOpeningTime> openingHour, String serviceHourType, Boolean validForNow, OffsetDateTime validFrom, OffsetDateTime validTo, List<LocalizedValue> additionalInformation) {
+    ServiceHour result = new ServiceHour();
+    result.setAdditionalInformation(additionalInformation);
+    result.setIsClosed(isClosed);
+    result.setOpeningHour(openingHour);
+    result.setServiceHourType(serviceHourType);
+    result.setValidForNow(validForNow);
+    result.setValidFrom(validFrom);
+    result.setValidTo(validTo);
+    return result;
+  }
+
+  protected List<VmOpenApiWebPageWithOrderNumber> createPtvInWebPages(String language, String url, String value) {
+    VmOpenApiWebPageWithOrderNumber result = new VmOpenApiWebPageWithOrderNumber();
+    result.setLanguage(language);
+    result.setOrderNumber("0");
+    result.setUrl(url);
+    result.setValue(value);
+    return Arrays.asList(result);
+  }
+
+  protected List<V7VmOpenApiAddressWithMovingIn> createPtvInAddressAbroad(List<VmOpenApiLanguageItem> locationAbroad) {
+    return createPtvInAddress("Location", PtvAddressSubtype.ABROAD.getPtvValue(), null, null, null, Collections.emptyList(), locationAbroad);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Location" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)